components/kyma-environment-broker/internal/error/temporary_error_test.go

```diff
@@ -15,9 +14,9 @@ func TestTemporaryError(t *testing.T) {
 	err3 := NewTemporaryError("some error: %s", fmt.Errorf("argErr"))
 
 	// when
-	e1 := errors.Wrapf(err1, "wrap err %s", "arg1")
-	e2 := AsTemporaryError(err2, "wrap err %s", "arg1")
-	e3 := errors.Wrapf(err3, "wrap err %s", "arg1")
+	e1 := fmt.Errorf("wrap err %s: %w", "arg1", err1)
+	e2 := fmt.Errorf("wrap err %s: %w", "arg1", err2)
```

Comment:
seems this e2 is not same as previous `e2 := AsTemporaryError(err2, "wrap err %s", "arg1")`..

Reply:
good catch, can you please take a look @tillknuesting?